### ISSUE TITLE
Not working in React 18

### ISSUE THREAD

**Issue by ASWIN-P-INFOPRISM** [appleple/react-modal-video#83]
react-modal-video is not getting installed in React 18, so kindly update react-modal-video to work in latest version of React.

**Comment by abdelrahman-muntaser** [comment]
Any solution yet for this problem ?

**Comment by bryanpsd** [comment]
Is this being worked on?

**Comment by matteo-dirollo** [comment]
I have the same issue. Any news?

**Comment by MosiurRrahman** [comment]
Any solution yet for this problem ?

**Comment by V3ntus** [comment]
This works in React 18

**Comment by ericaig** [comment]
It doesn't work for *React 18*

```
ericaig@EricAig-McBookPro moon-landing % npm i react-modal-video                                                        
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: moon-landing@0.1.0
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"18.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.2.0 || ^17.0.0" from react-modal-video@1.2.10
npm ERR! node_modules/react-modal-video
npm ERR!   react-modal-video@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/ericaig/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ericaig/.npm/_logs/2022-12-02T11_40_52_620Z-debug-0.log
```

**Comment by shantovai024** [comment]
not working on react 18.2.0

**Comment by V3ntus** [comment]
Workaround: install with --legacy-peer-deps

**Comment by uidev1116** [comment]
This issues was resolved by #81.
